invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/requests.js

```diff
@@ -142,8 +142,9 @@ export function RequestsResultsGridItemTemplate({ result, index }) {
 }
 
 export function RequestsResultsItemTemplate({ result, index }) {
-  const createdDate = new Date(result.created);
-  const differenceInDays = timestampToRelativeTime(createdDate.toISOString());
+  const timestampToRelativeTime = (timestamp) =>
```

Comment:
Please avoid putting this inside cmps or `render` methods, the func will be recreated each time.
Move this back to `utils.js`.